src/transmitter/payload.ts

```diff
@@ -134,3 +138,44 @@ export function constructWorkloadEventsPolicy(
     agentId: config.AGENT_ID,
   };
 }
+
+const workloadKindMap = {
+  deployment: 'Deployment',
+  replicaset: 'ReplicaSet',
+  statefulset: 'StatefulSet',
+  daemonset: 'DaemonSet',
+  job: 'Job',
+  cronjob: 'CronJob',
+  replicationcontroller: 'ReplicationController',
+  deploymentconfig: 'DeploymentConfig',
+  pod: 'Pod',
+};
+export function constructRuntimeData(
+  runtimeResults: IRuntimeImage[],
+): IRuntimeDataPayload {
+  const filteredRuntimeResults = runtimeResults.reduce((acc, runtimeResult) => {
+    if (!isExcludedNamespace(runtimeResult.namespace)) {
+      runtimeResult.workloadKind =
+        workloadKindMap[runtimeResult.workloadKind.toLowerCase()];
```

Comment:
Wonder if this should abort early and log an error if `workloadKindMap[runtimeResult.workloadKind.toLowerCase()]` returns undefined (in case that specific workload kind is malformed - we don't want to send bad data to the upstream)